src/generator/generate.ts

```diff
@@ -21,7 +21,8 @@ export const generateRuntimeAndEmit = (dmmf: DMMF.Document, settings: Settings.G
 
   const declarationSourceFile = ModuleGenerators.TS.createModule(dmmf, settings)
 
-  if (settings.data.output.directory === 'default') {
+  // comment this code because we generate only in on path
```

Comment:
We need to figure out a way to support ESM.
My recommendation is we generate a CJS by default, as I found in the Prisma Client, and the user can change this with custom settings.
We were only generating a CJS if the user selected a custom path to generate code in.

Reply:
The Prisma Client allows users to set a custom output directory. Do we intend to break from that pattern in this PR?

Reply:
IIUC we're formally revisiting ESM support later so not worrying about it now seems OK.

Reply:
@jasonkuhrt yep, looking to formally support that soon https://github.com/prisma/nexus-prisma/issues/200#issuecomment-1112384600